content/archive/verify-receipts/index.md

```diff
@@ -29,25 +29,33 @@ Receipts for Public Events can be obtained by any authenticated API request.
 
 ## What is in a receipt?
 
-The Receipts API is provided as an integration with emerging standards driven by [Supply Chain Integrity, Transparency, and Trust (SCITT)](https://www.datatrails.ai/what-is-scitt-and-how-does-datatrails-help/).
+This page discusses receipts in the context of verifying events recorded on the DataTrails platform.
 
-Regardless of how the standards evolve, any receipt you obtain today will remain valid proof of posting for the Event.
+For discussion of how DataTrails supports SCITT Transparent Statements and their corresponding SCITT Receipts see [Supply Chain Integrity, Transparency, and Trust (SCITT)](https://www.datatrails.ai/what-is-scitt-and-how-does-datatrails-help/).
```

Comment:
https://www.datatrails.ai/what-is-scitt-and-how-does-datatrails-help/ is 404

Reply:
Found it, it still references rkvst in the URL: https://www.datatrails.ai/what-is-scitt-and-how-does-rkvst-help/

Reply:
```suggestion
For a discussion of how DataTrails supports SCITT Transparent Statements and their corresponding SCITT Receipts see [Supply Chain Integrity, Transparency, and Trust (SCITT)](https://www.datatrails.ai/what-is-scitt-and-how-does-rkvst-help/).
```

Reply:
punting the RKVST to DataTrails url fix to a different PR